package.json

```diff
@@ -60,12 +62,16 @@
   },
   "scripts": {
     "build": "rollup -c",
+    "compile": "tsc --pretty",
+    "start": "live-server --port=5309 --open=src",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "test": "jest --passWithNoTests",
     "test:watch": "jest --watch",
     "prepublish": "npm run lint && npm run build",
     "release": "npm run prepublish && np",
-    "watch": "rollup -c --watch"
+    "watch": "rollup -c --watch",
+    "audit:ci": "audit-ci report-type summary",
+    "audit:ci:high": "audit-ci --high"
```

Comment:
What's the difference between these 2 scripts?

Reply:
good point.  I changed it as they both produce the same report summary where there are no high vulnerabilities.  Thanks